contracts/standard/arbitration/MultipleArbitrableTokenTransactionWithFee.sol

```diff
@@ -0,0 +1,478 @@
+/**
+ *  @authors: [@remedcu]
+ *  @reviewers: []
+ *  @auditors: []
+ *  @bounties: []
+ *  @deployments: []
+ *  @tools: [MythX]
+ */
+
+/** @title Multiple Arbitrable ERC20 Token Transaction with Fee
+ *  This is a contract for multiple arbitrated token transactions which can be reversed by an arbitrator.
+ *  This can be used for buying goods, services and for paying freelancers.
+ *  Parties are identified as "sender" and "receiver".
+ *  A fee is associated with each successful payment to receiver.
+ */
+
+pragma solidity ^0.4.24;
+
+import "./Arbitrator.sol";
+import "./IArbitrable.sol";
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+
+contract MultipleArbitrableTokenTransactionWithFee is IArbitrable {
+
+    // **************************** //
+    // *    Contract variables    * //
+    // **************************** //
+
+    uint8 constant AMOUNT_OF_CHOICES = 2;
+
+    enum Party {Sender, Receiver}
+    enum Status {NoDispute, WaitingSender, WaitingReceiver, DisputeCreated, Resolved}
+    enum RulingOptions {NoRuling, SenderWins, ReceiverWins}
+
+    struct Transaction {
+        address sender;
+        address receiver;
+        uint amount;
+        ERC20 token;
+        uint timeoutPayment; // Time in seconds after which the transaction can be automatically executed if not disputed.
+        uint disputeId; // If dispute exists, the ID of the dispute.
+        uint senderFee; // Total arbitration fees paid by the sender.
+        uint receiverFee; // Total arbitration fees paid by the receiver.
+        uint lastInteraction; // Last interaction for the dispute procedure.
+        Status status;
+    }
+
+    address public feeRecipient; // Address which receives a % of receiver payment.
```

Comment:
That's basis points, not %.
% -> share

Reply:
Corrected in https://github.com/kleros/kleros-interaction/pull/325